README.md

```diff
@@ -166,6 +174,42 @@ To create a `standard.x2` (or other non-default types) instance type, the `plan`
 managed_kafka.sh --create mykafka --plan standard.x2
 ```
 
+### Deploying to multiple clusters/zones
+In order to allow clients to deploy Kafka clusters to multiple clusters and/or zones, the following parameters must be customized:
+
+   * KUBE_CONFIG in `kas-fleet-manager-secrets-template-params`
```

Comment:
This is needed to be customizable only when running in standalone mode.
When using OCM, this won't be needed.
/cc @MikeEdgar